drivers/nrf24l01p_ng/include/nrf24l01p_ng_constants.h

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2019 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+/**
+ * @ingroup drivers_nrf24l01p_ng
+ * @{
+ *
+ * @file
+ * @brief   Constants from the datasheet of the NRF24L01+ (NG) transceiver
+ *
+ * @author  Fabian Hüßler <fabian.huessler@ovgu.de>
+ */
+#ifndef NRF24L01P_NG_CONSTANTS_H
+#define NRF24L01P_NG_CONSTANTS_H
+
+#include "nrf24l01p_ng.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Maximum width of a payload, restricted by
+ *          a FIFO size of 32 bytes
+ */
+#define NRF24L01P_NG_MAX_PAYLOAD_WIDTH      (32)
+
+/**
+ * @brief   Maximum number of retransmissions, if
+ *          ESB is used as protocol
+ */
+#define NRF24L01P_NG_MAX_RETRANSMISSIONS    (15)
+
+/**
+ * @brief   Base frequency
+ */
+#define NRF24L01P_NG_BASE_FRQ_MHZ           (2400)
+
+/**
+ * @brief   Maximum supported frequency
+ */
+#define NRF24L01P_NG_MAX_FRQ_MHZ            (2525)
+
+/**
+ * @brief    Number of supported channels
+ *
+ * frequency range: 2400 - 2525 Mhz
+ * occupied bandwidth for each channel:
+ *  250 kbps: < 1Mhz
+ *  1000 kbps: < 1Mhz
+ *  2000 kbps: < 2Mhz
+ *
+ * -> spaces between each channel´s
+ *    base frequency must be at least
+ *    2 Mhz, to be sufficient for all
+ *    data rates.
+ */
+#define NRF24L01P_NG_NUM_CHANNELS           (NRF24L01P_NG_MAX_FRQ_MHZ - \
+                                            NRF24L01P_NG_BASE_FRQ_MHZ)
+
+/**
+ * @brief   Reset value of TX_ADDR register
+ */
+#define NRF24L01P_NG_DEFAULT_TX_ADDR        { 0xE7, 0xE7, 0xE7, 0xE7, 0xE7 }
+
+/**
+ * @brief   Reset value of RX_ADDR_P0 register
+ */
+#define NRF24L01P_NG_DEFAULT_ADDR_P0        { 0xE7, 0xE7, 0xE7, 0xE7, 0xE7 }
+
+/**
+ * @brief   Reset value of RX_ADDR_P1 register
+ */
+#define NRF24L01P_NG_DEFAULT_ADDR_P1        { 0xC2, 0xC2, 0xC2, 0xC2, 0xC2 }
```

Comment:
Shouldn't those be obtained through `luid_get()`?
When I flash two boards they will both end up with `E7:E7:E7:E7:E7`

Reply:
I wanted to give the opportunity to set the address by hand. But you are right, I should set `NRF24L01P_NG_PARAM_RX_P0` and `NRF24L01P_NG_PARAM_RX_P1` to `NRF24L01P_NG_L2ADDR_AUTO`.

Thinking about you comments, now I want to do:
```C
        /* for pipe 0 , if not auto */
        luid_base(dev->params.urxaddr.rxaddrpx.rx_p0, addr_size);
        luid_get(&dev->params.urxaddr.rxaddrpx.rx_p0[addr_size - 1] , 1);

        /* for pipe 1, if not auto */
        luid_base(dev->params.urxaddr.rxaddrpx.rx_p1, addr_size);
        luid_get(&dev->params.urxaddr.rxaddrpx.rx_p1[addr_size - 1] , 1);

        /* pipe 2 and so on for pipe 3, 4, 5*/
        if (dev->params.urxaddr.rxaddrpx.rx_p2 != NRF24L01P_NG_ADDR_UNDEF) {
            luid_get(&dev->params.urxaddr.rxaddrpx.rx_p2 , 1);
        }
```

Reply:
Why call `luid_base()` *and* `luid_get()`?
The latter is just a wrapper for the former - why is calling `luid_get(dev->params.urxaddr.rxaddrpx.rx_p1, addr_size)` not enough?

Reply:
If I just call `luid_get()` for both p0 and p1, I get:
```
2020-05-04 14:15:27,151 #       Rx address p0:        77:24:26:11:12
2020-05-04 14:15:27,154 #       RX address p1:        76:24:26:11:12
```

The LSByte for both is 12, which is not allowed.
"The LSByte must be unique for all six pipes" [datasheet p.38]
Or is this not what you mean?

Reply:
Ah I didn't know there were restrictions on the format of the address. 
In this case your solution is of course correct - please add a comment with that quote from the data sheet.